msg/templates/urtps/microRTPS_transport.cpp

```diff
@@ -117,7 +119,11 @@ ssize_t Transport_node::read(uint8_t *topic_ID, char out_buffer[], size_t buffer
 		int errsv = errno;
 
 		if (errsv && EAGAIN != errsv && ETIMEDOUT != errsv) {
+#ifndef PX4_ERR
```

Comment:
Instead of all these ifdefs, something like this on top would be better:
```
#if __has_include("px4_log.h") && __has_include("px4_time.h")
#include <px4_log.h>
#include <px4_time.h>
#define log_err(fmt, ...) PX4_ERR(fmt, ##__VA_ARGS__)
#else
#define log_err(fmt, ...) printf(fmt "\n", ##__VA_ARGS__)
#endif
```